src/modules/verifiable-credentials/types/status-list.types.ts

```diff
@@ -0,0 +1,57 @@
+import { EIP712Proof } from '@ew-did-registry/credentials-interface';
+import { IProof } from '@sphereon/pex';
+
+export class StatusList2021UnsignedCredential {
+  '@context': Array<string | Record<string, unknown>>;
+  id: string;
+  type: string[];
+  issuer: string;
+  issuanceDate: string;
+  credentialSubject: {
+    id: string;
+    type: string;
+    statusPurpose: string;
+    encodedList: string;
+  };
+}
+
+export class StatusList2021Credential {
```

Comment:
@Harasz Should we move `StatusList2021Credential` to `ew-did-registry`, as it would be consumed in verification as well?